tests/scripts/thread-cert/Cert_5_8_02_KeyIncrement.py

```diff
@@ -85,56 +85,55 @@ def verify(self, pv):
 
         # Step 1: The DUT must start the network using
         # thrKeySequenceCounter = 0
-        _lpkts = leader_pkts.filter_mle_cmd(consts.MLE_ADVERTISEMENT).must_next()
+        _lpkts = leader_pkts.filter_mle_cmd(MLE_ADVERTISEMENT).must_next()
         self.assertTrue(_lpkts.wpan.aux_sec.key_source == 0)
 
         # Step 2: Verify that the topology described above is created.
         # MLE Auxiliary security header shall contain Key Source = 0,
         # KeyIndex = 1, KeyID Mode = 2
-
-        leader_pkts.filter_mle_cmd(consts.MLE_CHILD_ID_RESPONSE).must_next()
+        leader_pkts.filter_mle_cmd(MLE_CHILD_ID_RESPONSE).must_next()
         _lpkts = leader_pkts.range(leader_pkts.index)
         _rpkts = router_pkts.range(leader_pkts.index)
 
-        _rpkts.filter_mle_cmd(consts.MLE_ADVERTISEMENT).must_next()
-        [_,i] = _rpkts.index
-        self.assertTrue(_rpkts[i-1].wpan.aux_sec.key_index == 1 and
-                        _rpkts[i-1].wpan.aux_sec.key_id_mode == 2 and
-                        _rpkts[i-1].wpan.aux_sec.key_source == 0)
+        _rpkts.filter_mle_cmd(MLE_ADVERTISEMENT).must_next()
+        [_, i] = _rpkts.index
+        self.assertTrue(_rpkts[i - 1].wpan.aux_sec.key_index == 1 and
+                        _rpkts[i - 1].wpan.aux_sec.key_id_mode == 2 and
+                        _rpkts[i - 1].wpan.aux_sec.key_source == 0)
```

Comment:
```suggestion
        _rpkts.filter_mle_cmd(MLE_ADVERTISEMENT).must_next().must_verify("""
            wpan.aux_sec.key_index == 1 and
            wpan.aux_sec.key_id_mode == 2 and 
            wpan.aux_sec.key_source == 0
        """)
```

Usually, we don't need to get the `index` manually. 

`assertTrue` works, but better call `must_verify`.